docker-entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# Docker entrypoint script to replace s6-overlay functionality
+
+# Set default values
+PUID=${PUID:-911}
+PGID=${PGID:-911}
+
+# Update user and group IDs
+echo "Setting up user 'abc' with UID: $PUID, GID: $PGID"
+groupmod -o -g "$PGID" abc
+usermod -o -u "$PUID" abc
+
+# Display sponsorship message
+echo "
+====================================================
+To support this project, please consider sponsoring.
+https://github.com/sponsors/mandarons
+https://www.buymeacoffee.com/mandarons
+
+User UID:    $(id -u abc)
+User GID:    $(id -g abc)
+===================================================="
+
+# Display build version if available
+if [ -f /build_version ]; then
+    cat /build_version
+fi
+
+# Create necessary directories
+mkdir -p /icloud /config/session_data
+
+# Set ownership
+chown -R abc:abc /app /config /icloud
```

Comment:
first check the owner of these directories. If they are not abc:abc (check user id and group id as well), then only do recursive chown.